From 79651f6be03752ecaefc9c77e2460e1763f6c0ea Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Fri, 1 Jul 2016 14:24:27 -0400
Subject: [PATCH 29/55] Minor man page changes (#1232168)

Minor wording changes made in the Architecture section after placing the
initial draft up for public comment.

Related: rhbz#1232168
---
 grubby.8 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/grubby.8 b/grubby.8
index 690676858ca..fdf2766fe9a 100644
--- a/grubby.8
+++ b/grubby.8
@@ -29,12 +29,12 @@ Legacy \fBs390\fR and the current \fBs390x\fR architectures
 and their \fBzipl\fR bootloader are fully supported.
 
 Support for \fByaboot\fR has been deprecated as all ppc architecture
-hardware since the Power8 system uses \fBgrub2\fR or petitboot
+hardware since the Power8 uses \fBgrub2\fR or petitboot
 which both use the grub2 configuration file format.
 
 Legacy bootloaders \fBLILO\fR, \fBSILO\fR, and \fBELILO\fR
-are deprecated in favor of previously mentioned bootloaders. The
-\fBSILO\fR bootloader should also be considered unsupported.
+are deprecated and no longer receiving active support in favor of
+previously mentioned bootloaders.
 
 .SS Default Behavior
 
-- 
2.17.1

